docs/websites/README.md

```diff
@@ -73,6 +71,10 @@ After this, you can access the website at the following ports:
 1. Run `./build.sh clean`.
 
 #### Running tests
+You might need to install the broken-link-checker first
+
+`npm install broken-link-checker`
+
```

Comment:
This runs in the container now doesn't it? Shouldn't need it locally?

Reply:
For some reason, npm kept failing on my Windows machine at home because it couldn't find broken-link-checker